lib/admin.js

```diff
@@ -182,18 +180,18 @@ Admin.prototype.ping = function(options, callback) {
  * @return {Promise} returns Promise if no callback passed
  */
 Admin.prototype.addUser = function(username, password, options, callback) {
-  var self = this;
-  var args = Array.prototype.slice.call(arguments, 2);
+  const args = Array.prototype.slice.call(arguments, 2);
   callback = typeof args[args.length - 1] === 'function' ? args.pop() : undefined;
 
   options = args.length ? args.shift() : {};
   options = options || {};
```

Comment:
generally if you have an `options = options || {};` and a subsequent need to shallow copy them, you can wrap them together in a single `options = Object.assign({}, options);`